assets/scss/_content.scss

```diff
@@ -46,8 +46,6 @@
         width: 20.0rem;
         text-align: right;
         margin-right: 3.0rem;
-        text-align: right;
-        margin-right: 3.0rem;
```

Comment:
Why removing these lines?

Reply:
duplicate of the two lines above it. most probably copy/paste issue.

Reply:
Ooh! I did not notice these 2 lines above.